### PR TITLE
🔻 sales - opacity (animation)

### PR DESCRIPTION
**Pull request by neugomonov**

### CODE REVIEW

**Review by pauket — APPROVED** [review]

File: client/src/hoc/Wrapper.tsx [hunk 1]
```diff
@@ -26,7 +27,9 @@ const Wrapper = (Component: NextComponentType) => {
       <ThemeProvider theme={theme}>
         <CssBaseline />
         <Layout>
-          <Component {...props} />
+          <AnimatePresence exitBeforeEnter>
+            <Component {...props} />
+          </AnimatePresence>
```
Comment (neugomonov) [inline client/src/hoc/Wrapper.tsx:32]: nah not working
this wrapper is really bad design
gotta find another workaround for this redux crap to start working properly in the _app

File: client/src/pages/admin/index.tsx [hunk 2]
```diff
@@ -49,7 +50,23 @@ const Admin = () => {
...
+        appearing: { opacity: 0 },
+        visible: {
+          opacity: 1,
+          transition: {
+            duration: 1,
+          },
+        },
+      }}
+      exit={{ opacity: 0 }}
+    >
```
Comment (neugomonov) [inline client/src/pages/admin/index.tsx:69]: refactor as reusable just the key as an arg

Comment (neugomonov) [inline client/src/pages/admin/index.tsx:69]: refactor as reusable just the key as an arg

File: client/src/hoc/Wrapper.tsx (hunk 1 above)
Comment (neugomonov) [inline client/src/hoc/Wrapper.tsx:32]: nah not working
this wrapper is really bad design
gotta find another workaround for this redux crap to start working properly in the _app

File: client/src/pages/dashboard/index.tsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ import Header from "components/Header";
 import MotionBox from "components/motion/MotionBox";
 import OverviewChart from "components/OverviewChart";
 import StatBox from "components/StatBox";
+import { AnimatePresence, motion } from "framer-motion";
```
Comment (neugomonov) [inline client/src/pages/dashboard/index.tsx:22]: unused

**Review by neugomonov — COMMENTED** [review]
ungood rectify

**Pull request merged**